test/HelloWorld.test.js

```diff
@@ -0,0 +1,14 @@
+const { assert } = require('chai');
+
+describe('Array', () => {
+  describe('#indexOf()', () => {
+    it('should return -1 when the value is not present', function() {
+      assert.equal(-1, [1,2,3].indexOf(4));
+    });
+  });
+});
+
+
+
+
+
```

Comment:
Remove all the lines from 10 to 14. After that, at the end of line 9 press "Enter". File should have only 10 lines, being the 10th line empty

Reply:
Fixed, thanks for the comment.